### ISSUE TITLE
Make line for user guide more noticeable

### ISSUE THREAD

**Issue by ppwadhwa** [pyviz-topics/examples#153]
On the main page for https://examples.pyviz.org/, there is a line directing the user to the user guide which explains how to use anaconda project for all the examples, but that link for the user guide isn't very noticeable.  As a new user, my eyes are drawn to all the pictures and I immediately want to click on a picture, I don't notice the sentence at the top at all.  I think the user guide sentence should stand out more to get attention.  Another solution would be to have a link to the user guide on each example so hopefully a person would notice it there when looking at specific examples.

**Comment by jbednar** [comment]
I agree, most people won't see that!  I'd be happy with putting that link into its own less-missable section at the top, drawing attention to itself and explaining a bit more what cool things can be done with these projects and how the user guide shows you how to do those things. It would also be fine to have a link on each example, which the only way to catch someone who followed a direct link to a project, but that sounds like more work to set up.
